libpod/test_common.go

```diff
@@ -0,0 +1,116 @@
+package libpod
+
+import (
+	"encoding/json"
+	"net"
+	"path/filepath"
+	"reflect"
+	"time"
+
+	"github.com/containers/storage"
+	"github.com/cri-o/ocicni/pkg/ocicni"
+	"github.com/opencontainers/runtime-tools/generate"
+)
+
+// nolint
```

Comment:
Might benefit from a comment saying what specific lint failure should be ignored here, in case tools change or new warnings appear.